tests/test_open_tmux_project_folder.py

```diff
@@ -0,0 +1,64 @@
+import sublime
+import io
+from unittest import mock
+import snapshottest
+
+def mock_process_call(attrs):
+    process_mock = mock.Mock()
+    process_mock.configure_mock(**attrs)
+
+    return process_mock
+
+def mock_tmux_subproc_calls():
```

Comment:
Refactor to share these methods between files needed. Potentially rename this method as `mock_standard_tmux_subproc_events` or similar, to indicate that this is a base case and could be altered for future tests.